src/ExtensionsSample/Samples/SendGridSamples.cs

```diff
@@ -19,14 +19,15 @@ public static class SendGridSamples
         public static void ProcessOrder_Declarative(
             [QueueTrigger(@"samples-orders")] Order order,
             [SendGrid(
+                ApiKey = "Test",
```

Comment:
Yes, I meant to clean that up. The new E2E tests verify that it's optional and fallbacks work as expected.